### PR TITLE
feat(message-system): add pol warning banner

### PR DESCRIPTION
**Pull request by MiroslavProchazka**
Adding a message banner that will be displayed to all users with an activated MATIC account, informing them about the migration to POL. See this [Notion page](https://www.notion.so/satoshilabs/Polygon-updates-02e9dea80a704984bf59f37b35e7224e?pvs=4) for more info

Tracked in Notion [here](https://www.notion.so/satoshilabs/MATIC-migration-to-polygon-c92d473b049f4c13a8b50c4346062ada?pvs=4)

image

To Do:

- [x] Proofread translations to all languages
- [x] Get KB article url 
- [ ] Publish to dev environment and test it

### CODE REVIEW

File: suite-common/message-system/config/config.v1.json [hunk 1]
```diff
@@ -90,6 +90,63 @@
...
+                }
+            ],
+            "message": {
+                "id": "c8ca8958-7868-467f-9278-f072edc234ac",
+                "priority": 93,
+                "dismissible": true,
+                "variant": "info",
+                "category": "banner",
+                "content": {
+                    "en-GB": "Polygon PoS will migrate from MATIC to POL on September 4. Your balance will appear as MATIC until an update is available.",
```
Comment (sime) [inline suite-common/message-system/config/config.v1.json]:
```suggestion
                    "en-GB": "Polygon PoS will migrate from MATIC to POL on September 4, no action is required. Your balance will appear as MATIC until an update is available.",
```

File: suite-common/message-system/config/config.v1.json [hunk 2]
```diff
@@ -90,6 +90,63 @@
...
+                    "en": "Polygon PoS will migrate from MATIC to POL on September 4. Your balance will appear as MATIC until an update is available.",
```
Comment (sime) [inline suite-common/message-system/config/config.v1.json]:
```suggestion
                    "en": "Polygon PoS will migrate from MATIC to POL on September 4, no action is required. Your balance will appear as MATIC until an update is available.",
```

**Review by sime — APPROVED** [review]
Added 'no action is required', but this shouldn't block anything

File: suite-common/message-system/config/config.v1.json (hunk 1 above)
Comment (MiroslavProchazka) [reply suite-common/message-system/config/config.v1.json]: Solved in f451b7d85bdf86d94038afdccbc6c772152abd7a

File: suite-common/message-system/config/config.v1.json (hunk 2 above)
Comment (MiroslavProchazka) [reply suite-common/message-system/config/config.v1.json]: Solved in f451b7d85bdf86d94038afdccbc6c772152abd7a

**Comment by STew790** [comment]
**QA OK**
image

Tested on https://dev.suite.sldev.cz/suite-web/chore/msg-system-pol/web/

**Info:**
 - Suite version: web 24.9.0 (95f8989c855264fd9555152d5972d5fee232ad02)
 - Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: Trezor T3T1 2.8.1 regular (revision 632b9561559b7ab6824bb7eeac072874e07b7b82)
 - Transport: WebUsbTransport

**Review by komret — APPROVED** [review]

**Pull request merged**